### DISCUSSION TITLE
Displaying a Pcam feed on a surface (CCTV style)

### DISCUSSION THREAD

**Discussion by electroskeptic** [ramokz/phantom-camera#356] [Ideas]
Is it possible to put a Pcam viewport onto a plane so that it can be visible in-game?

**Comment by ramokz** [comment]
Short answer, no. Projecting a viewport onto a plane requires a separate `Camera3D`.

`PCams` are merely just `Node3Ds` with additional logic applied onto them that relays data, position etc., to a given `Camera3D`.
`PCams` in of themselves do not render anything.

**Reply by electroskeptic** [reply]
Thanks for the answer!
But if I add another Camera3D, Pcam setup will stop to work? Or will it allow that additional camera?

**Reply by ramokz** [reply]
You can add as many `Cameras2D/3Ds` to the scene as you want.

But `PCams` can only work with one Camera node at the minute, where if you attempt to add another `PCamHost`, which is what talks to the `PCams`, it will automatically remove the second `PCamHost` instance from the scene — a log will also appear in the Output tab saying why if you try it.

Having multiple `PCamHosts` in a scene will be covered in #26.
